crates/tests/agile_reference/tests/tests.rs

```diff
@@ -0,0 +1,15 @@
+use windows::core::AgileReference;
+use windows::Media::Control::GlobalSystemMediaTransportControlsSessionManager;
+
+#[test]
+fn test() {
```

Comment:
```suggestion
fn test() -> windows::core::Result<()> {
```

Should make the test more readable using `?` instead of `unwrap()`.

Reply:
OK, probably not all that much given the `spawn`...

Reply:
It should simplify it a bit. I could also use a `Result` in the thread.

I'll do that tomorrow.